recipes/dask-ms/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dask-ms" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 40a91034e53e2efb9b289eb5d9475a1957e1bd4dee707d133c3120990c5658d1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - dask
+    - numpy
+    - python-casacore
+    - setuptools
+    - xarray
+  run:
+    - python
+    - dask
+    - numpy
+    - python-casacore
+    - xarray
+
+test:
+  imports:
+    - daskms
+
+about:
+  home: https://github.com/ska-sa/dask-ms
```

Comment:
add a `license` field. It looks like a BSD-3-Clause (https://github.com/ska-sa/dask-ms/blob/master/LICENSE), but the `setup.py` meta says proprietary:

https://github.com/ska-sa/dask-ms/blob/master/setup.py#L45

Reply:
Thanks for reviewing the setup.py. I've updated it to reflect the BSD license in the pypi classifiers.

https://github.com/ska-sa/dask-ms/pull/104

Reply:
Note this change will only reflect in the upcoming dask-ms 0.2.4.

Reply:
Thanks both. I will update this recipe accordingly to see whether it passes the tests.

I am also happy to wait for the `0.2.4` release before we merge.